test/tests/multiapps/sub_cycling/master_sub_output.i

```diff
@@ -0,0 +1,68 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+  [./td]
+    type = TimeDerivative
+    variable = u
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  [../]
+  [./right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = 4
+  dt = 0.1
+
+  # Preconditioned JFNK (default)
+  solve_type = 'PJFNK'
+
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+[]
+
+[Outputs]
+  output_initial = true
+  exodus = true
+  print_linear_residuals = true
+  print_perf_log = true
+[]
+
+[MultiApps]
+  [./sub]
+    type = TransientMultiApp
+    app_type = MooseTestApp
+    execute_on = timestep_end
+    positions = '0 0 0'
+    input_files = sub.i
+    sub_cycling = true
+    output_sub_cycles = true
```

Comment:
Sorry "not understand" is strong. I meant I didn't understand that there
was another parameter that I had to set to not lose output. Bit me again.
On Tue, Feb 10, 2015 at 5:03 PM Derek Gaston <notifications@github.com>
wrote:

> In test/tests/multiapps/sub_cycling/master_sub_output.i
> <https://github.com/idaholab/moose/pull/4659#discussion_r24463946>:
>
> > +[Outputs]
> > +  output_initial = true
> > +  exodus = true
> > +  print_linear_residuals = true
> > +  print_perf_log = true
> > +[]
> > +
> > +[MultiApps]
> > +  [./sub]
> > +    type = TransientMultiApp
> > +    app_type = MooseTestApp
> > +    execute_on = timestep_end
> > +    positions = '0 0 0'
> > +    input_files = sub.i
> > +    sub_cycling = true
> > +    output_sub_cycles = true
>
> Why is this a parameter that only developers understand? It causes File
> output of the intermediary steps.
>
> Wait: @aeslaughter <https://github.com/aeslaughter> is this necessary to
> get console output of intermediate steps? If so... That is wrong. As per
> usual we always want console output of intermediate steps: this option is
> for doing file output of intermediate steps (which you normally DONT want
> unless you're debugging)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/4659/files#r24463946>.
>